WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -35,13 +41,23 @@ class LoginPrologueViewController: LoginViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        
         configureButtonVC()
         navigationController?.setNavigationBarHidden(true, animated: false)
     }
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
+        tracker.set(flow: .prologue)
```

Comment:
This should be wrapped by the `.unifiedWordPress` flag. It should not be tracking the old prologue screen / old prologue buttons.

Reply:
I understand.

Just one clarification, though.  If I wrap this in `.unifiedWordPress` we won't be seeing this in 15.6 builds.  Is that fine?

Reply:
Yes that's perfect. The new prologue buttons are in 15.7.

Reply:
Instead of removing the tracking, I disabled tracking for the prologue in the tracker's configuration.  This will be changed in 15.7 to follow the `.unifiedWordPress` feature flag.